### PR TITLE
read_json() should work for all valid JSON input [BW-428]

### PR DESCRIPTION
**Pull request by salonishah11**
Closes https://broadworkbench.atlassian.net/browse/BW-428

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,12 @@
 # Cromwell Change Log
 
+## 55 Release Notes
+
+### Bug Fixes
+
+* Fixed a bug where `read_json()` failed for `Array[_]` inputs. It should now work for all valid JSON data inputs.
```
Comment (aednichols) [inline CHANGELOG.md:7]: I feel like we revised the 54 release notes in the previous PR, maybe it got lost in git? If you chose not to implement the suggestion I'm not offended either 😄

**Review by aednichols — APPROVED** [review]
LGTM 👍

File: CHANGELOG.md (hunk 1 above)
Comment (salonishah11) [reply CHANGELOG.md:7]: Oh that is weird! I do remember committing your suggestion!

Comment (salonishah11) [reply CHANGELOG.md:7]: Oh I think I know what happened! So I committed your suggestion through Github, but then I had to make some other changes and when I rebased it against develop I forced pushed all changes without pulling the commit with your suggestion onto local branch. And hence that commit never made it to develop :(

I will update this CHANGELOG with your suggestion from previous PR.

**Review by rsasch — APPROVED** [review]

**Pull request merged**